elasticsearch/README.md

```diff
@@ -129,33 +129,29 @@ make
 
 ### Security
 
-A cluster with X-Pack security enabled
+A cluster with security enabled
 
-* Generate SSL certificates following the [official docs](https://www.elastic.co/guide/en/elasticsearch/reference/6.7/configuring-tls.html#node-certificates)
-* Create Kubernetes secrets for authentication credentials and certificates
+* Generate SSL certificates following the [official docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/configuring-tls.html#node-certificates)
+* Create Kubernetes secrets for authentication credentials and certificates. Replace `$YOUR_SECRET_PASSWORD` with your own password.
   ```
-  kubectl create secret generic elastic-credentials --from-literal=password=changeme --from-literal=username=elastic
+  kubectl create secret generic elastic-credentials --from-literal=password=$YOUR_SECRET_PASSWORD --from-literal=username=elastic
   kubectl create secret generic elastic-certificates --from-file=elastic-certificates.p12
   ```
 * Deploy!
   ```
   cd examples/security
-  make
+  helm upgrade --wait --timeout=600 --install --values ./security.yml elasticsearch ../../
   ```
 * Attach into one of the containers
 
   ```
-  kubectl exec -ti $(kubectl get pods -l release=helm-es-security -o name | awk -F'/' '{ print $NF }' | head -n 1) bash
+  kubectl exec -ti $(kubectl get --no-headers=true pods -l release=elasticsearch -o custom-columns=:metadata.name | head -n 1 ) bash
```

Comment:
` `  :scream:

Reply:
One step closer to being able to do basic operations without needing third party tooling!